web/thesauruses/java/11/exception_handling.json

```diff
@@ -0,0 +1,316 @@
+            "comment": "Here, `CustomException` is the new exception that you have defined"
+        },
+        "statement_to_catch_generic_exception": {
+            "name": "Statement to catch a generic exception",
+            "code": "catch(Exception e) {\n    statements\n}"
+        },
+        "statement_to_catch_specific_exception": {
+            "name": "Statement to catch a specific exception",
+            "code": "catch(CustomException e) {\n    statements\n}",
+            "comment": "Here, `CustomException` is the new exception that you have defined"
+        },
+        "statement_to_catch_multiple_exceptions": {
+            "name": "Statement to catch multiple types of exceptions",
+            "code": "catch(ExceptionType1 | ExceptionType2 | ExceptionType3 e) {\n    statements\n}"
+        },
+        "statement_to_break_out_if_caught_exception": {
+            "name": "Statement to break out of a block if an exception is caught",
+            "not-implemented": true
+        },
+        "statement_to_stop_program_if_caught_exception": {
+            "name": "Statement to stop the program if an exception is caught",
+            "code": "catch(CustomException e) {\n    System.exit(1)\n}",
+            "comment": "The exit command placed in the catch block will terminate the program if an exception of type `CustomException` occurs"
+        },
+        "statement_to_start_code_if_no_exceptions_caught": {
+            "name": "Statement to start code if no exceptions are caught",
+            "code": [
+                "bool success = true;",
+                "try {",
+                "    something();",
+                "} catch (Exception e) {",
+                "    success = false;",
+                "}",
+                "if (success) {",
+                "}"
+            ],
+            "comment": "Not natively possible, but we can use this hack to simulate Python's try-except-else"
+        },
+        "statement_to_start_cleaning_up_after_exception": {
+            "name": "Statement to start code ran after an exception",
+            "code": "finally {\n    statements\n}"
+        },
+        "order_to_call_all_statements": {
+            "name": "The order to call all statements in a block to catch exceptions",
+            "code": [
+                "bool success = true;",
+                "try {",
+                "    something();",
+                "} catch (Exception e) {",
+                "    success = false;",
+                "}",
+                "    if (success) {",
+                "}",
+                "finally {",
+                "}"
+            ]
+        },
+        "statement_to_end_trying_code": {
+            "name": "Statement to end a block of code to try ",
+            "not-implemented": true
+        },
+        "property_with_exception_code": {
+            "name": "Property within an exception object containing the error code",
+            "code": "cause"
```

Comment:
So this is more like `Exception.cause` instead of just `cause`?

Reply:
Hmm, more like `Exception.getCause()`. Nice catch, I've changed this part and added clarification.

Reply:
I was thinking you'd add the class in, but I guess saying it inherits from Throwable can work.